c/python/namedtuple.cc

```diff
@@ -132,4 +132,14 @@ onamedtuple::onamedtuple(const onamedtupletype& type) {
 }
 
 
+void onamedtuple::copy_v() {
```

Comment:
`copy_v()` is quite a mysterious name, how about `prepare_for_editing()` or `make_editable()`? And also add a comment why do we need to jump through these hoops: because in python a tuple can only be modified if its ref-count is 1.

Reply:
Sure, we can have a better name for it.